ceryx/nginx/lualib/router.lua

```diff
@@ -2,10 +2,16 @@ local host = ngx.var.host
 local is_not_https = (ngx.var.scheme ~= "https")
 local cache = ngx.shared.ceryx
 
+function route(source, target)
+    ngx.var.container_url = target
+    ngx.log(ngx.INFO, "Routing request for " .. source .. " to " .. target .. ".")
```

Comment:
I would consider this `DEBUG` logging.

Reply:
I had this debate with myself as well. I intentionally ended up logging this information as `INFO` because it's the most critical information about each request. I am basically doing the same thing when no target can be found.